dbbuild/sources/byrskabishop2022/util.py

```diff
@@ -46,6 +46,9 @@ def __init__(self, vcf, refr_fasta):
 
     def get_haplotypes(self, chrom, positions):
         reference = self.get_reference_haplotype(chrom, positions)
+        if len(reference) != len(positions):
+            message = f"variant count mismatch: {len(reference)} vs {len(positions)}"
+            raise ValueError(message)
```

Comment:
Added the following code to the build procedure to check for similar issues in the future.